reader-shp/src/main/java/com/graphhopper/reader/shp/GraphhopperShp.java

```diff
@@ -0,0 +1,26 @@
+package com.graphhopper.reader.shp;
+
+import java.util.HashSet;
+
+import com.graphhopper.GraphHopper;
+import com.graphhopper.reader.DataReader;
+import com.graphhopper.reader.shp.OSMShapeFileReader.EdgeAddedListener;
+import com.graphhopper.storage.GraphHopperStorage;
+
+public class GraphhopperShp extends GraphHopper {
```

Comment:
Changed in the next commit. Probably not gramatically correct though! OSM is an acronymn, SHP isn't. Licence file added.